.github/ISSUE_TEMPLATE/5_KIT_template.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Propose KIT
+labels: kit
+---
+
+# Overview
+
+## Description
+<!-- Please provide a short description -->
+
+## Impact
+<!-- Please name the impact -->
+- Impact 1
+
+## KIT Team
+
+### Contributor
+<!-- names are already needed for open planning. Please also assign the contributor to this issue to ensure better clarity and accountability. -->
+- Contributor 1
+- Contributor 2
+
+### Committer
+<!-- names are already needed for open planning. Please also assign the committer to this issue to ensure better clarity and accountability -->
+- Committer 1
+- Committer 2
+
+## Standard Relevance
+
+- [ ] This feature aligns with our current set of standards
```

Comment:
Wouldn't it be of interest which standards will be reflected in the KIT. This could be part of a review, if e.g., a standard has been forgotten and then the tick here on this line might be wrong.

Reply:
Hi @lgblaumeiser, I see your point. But as each KIT always centers around a specific set of standards (which usually does not change) I don't think it provides additional value to add them et al. At least not at the stage of the planning. In the same time, we potentially do not know in advance whether the feature will actually align with the current set of standards and some aspects may be missed during implementation. 

Potentially, it would make more sense to add this check to the PR template where the KITs are actually implemented and not during the planning where it is only a hypothetical compliance?

Reply:
Sounds good to me, thanks @jSchuetz88